libkbfs/disk_block_cache_helper.go

```diff
@@ -15,13 +15,20 @@ type diskBlockCacheEntry struct {
 	ServerHalf kbfscrypto.BlockCryptKeyServerHalf
 }
 
+// Wrap time.Time so that go-codec falls back to using
+// time.Time.MarshalBinary instead of its new msgpack timestamp
+// extension encoding.
+type legacyEncodedTime struct {
```

Comment:
What if you just do `legacyEncodedTime time.Time`?

Reply:
Tried that, didn't work. I suspect because the way that go-codec uses reflection is blind to type aliases.

Also, it's more annoying because to call any methods on `legacyEncodedTime` you'd first have to cast to `time.Time`.